docs/decisions/0013-microsoft-fhir-converter.md

```diff
@@ -0,0 +1,105 @@
+
+Pros:
+- The same conversion tool we are already familiar with
+- Support for both Hl7v2 and C-CDA
+- Can be installed on Windows, MacOS, and Linux systems.
+- Open source and actively maintained by Microsoft.
+
+Cons:
+- Installation requires building from source.
+- Written in C# with .NET framework making integration with PHDI's primarily Python based work challenging.
+
+### LinuxForHealth FHIR Converter
+
+The [LinuxForHealth FHIR Converter](https://github.com/LinuxForHealth/hl7v2-fhir-converter) is an open source Hl7v2 to FHIR conversion utility written in Java.
+
+Pros:
+- Supports Hl7v2
+- Open source 
+- Can be built on any system with a JVM.
+
+Cons:
+- Does not support C-CDA
+- Installation process is non-trivial
+
+### Report Stream Fhirengine
+
+The Report Stream project is developing a FHIR conversion tool called [fhirengine](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/src/main/kotlin/fhirengine). It is open source, written in Kotlin, and supports bi-directional Hl7v2 and FHIR conversion.
+
+Pros:
+- Supports Hl7v2
+- Supports conversion from FHIR back to Hl7v2.
+- Can be built on any system with a JVM.
+- Designed, built, and maintained by another project with in PRIME.
+
+Cons:
+- Does not support C-CDA.
+- Installation process is non-trivial.
+ 
+
+### CDC eCR Team FHIR Converter
+
+The Electronic Case Reporting team at CDC is developing a FHIR conversion tool.
+
+Pros:
+- Supports C-CDA
+- Developed by another team at CDC.
+
+Cons:
+- We believe that it does not support Hl7v2, but have not be able to confirm this.
+- The eCR team has not be responsive to setting up a meeting to dsicuss their converter.
+
+### GCP Healthcare Data Harmonization
+
+The [GCP Healthcare Data Harmonization](https://github.com/GoogleCloudPlatform/healthcare-data-harmonization) engine offers tools for conversion from Hl7v2 and C-CDA to FHIR.
+
+Pros:
+- Supports Hlv2
+- Supports C-CDA
+- Open source
+
+Cons:
+- Documentation and examples do not seem to cover C-CDA conversion although reviewing the code base shows that C-CDA conversion is supported.
+- Installation has several dependencies and is non-trivial
+
+
+## Decision Outcome
+
+**Microsoft FHIR Converter** 
+
+Our need for conversion of both Hl7v2 and C-CDA eliminates all options except the Microsoft FHIR Converter and GCP Healthcare Data Harmonization engine. We have elected to move forward with Microsoft FHIR Converter for the following reasons:
+
+- As a team we are already familiar with the Microsoft FHIR Converter.
+- Although non-trivial the installation process is simpler than the GCP Healthcare Data Harmonization engine and has fewer dependencies.
+- After installation the CLI makes the tool easy to use.
+- We have been able to build a containerized application around this converter that makes its functionality availabe via an HTTP API and can be run on any platform that supports Docker containers.
```

Comment:
We shouldn't list this as part of our decision factors. We very likely could have done the same with Google's option. I would replace this with a bullet point highlighting that Microsoft's solution comes out of the box with 57 Hl7v2 templates and 9 C-CDA templates.